### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by kunicmarko20**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
     "name": "sonata-project/twig-extensions",
-    "type": "symfony-bundle",
-    "description": "Symfony twig extensions",
+    "type": "library",
```
Comment (kunicmarko20) [inline composer.json]: not sure about this, don't merge.

Comment (OskarStark) [reply composer.json]: It is a lib AFAIK

Comment (kunicmarko20) [reply composer.json]: yes, I just didn't know if I was going to make it a php lib or a symfony bundle, at the end it is lib with a Symfony Bridge, so this is ok.

**Comment by greg0ire** [comment]
Thanks @kunicmarko20 !

**Pull request merged**